baybe/kernels/priors/basic.py

```diff
@@ -0,0 +1,16 @@
+"""Priors that can be used for kernels."""
+from attrs import define, field
+from attrs.validators import gt
+
+from baybe.kernels.priors.base import Prior
+
+
+@define
```

Comment:
should we consider frozen and or slotted?

Reply:
Oh yes, absolutely! Both for the kernels and the priors it's missing!

Reply:
No: The base classes are frozen (this is explicitly declared), and slots is true by default. Also, according to the attrs documentation, the child classes are thus also frozen (see point 5):
![image](https://github.com/emdgroup/baybe/assets/84016733/a3e3b992-c491-48c9-bac3-a6abbbdb9d3c)
So it's not missing?

Reply:
Yes, the slots is fine, I know. But thanks for the info on inherited frozenness (<- is that a word 🤔), wasn't aware of that. But still, I think at this point we should add it for consistency reasons, just because we have it added it explicitly for ALL OTHER subclasses in the codebase. So in principle we can omit it, but if we do so, this should rather happen in one of the general cleanup PRs in the future (and it's not really high priority). So would favor consistency for now ...

Reply:
Thy wish shall be granted.

Reply:
Thanks. In fact, it has another advantage: in case we ever need to remove frozenness for the base class, we still have it for the subclasses and can control it case by case. But that's rather minor. Thanks for changing ⭐